Stripe/STPEphemeralKey.m

```diff
@@ -80,7 +86,8 @@ - (BOOL)isEqualToResourceKey:(STPEphemeralKey *)other {
     return [self.stripeID isEqualToString:other.stripeID] 
         && [self.secret isEqualToString:other.secret]
         && [self.expires isEqual:other.expires]
-        && [self.customerID isEqual:other.customerID];
+        && [self.customerID isEqual:other.customerID]
```

Comment:
I think `[nil isEqual: nil]` returns false, and so this breaks when you make `customerID` `nullable`